runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -9806,6 +9806,36 @@ static TR::Register *inlineSinglePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT_FATAL(node->getNumChildren() == 3, "There should be exactly 3 children in the node of inlineFPTrg1Src3");
+
+   TR::DataType type = node->getDataType();
+   TR_ASSERT_FATAL(type == TR::Float || type == TR::Double, "Node of inlineFPTrg1Src3 should be either TR::Float or TR::Double");
+
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Node *thirdChild = node->getThirdChild();
+   TR::Register *src1Register = cg->evaluate(firstChild);
+   TR::Register *src2Register = cg->evaluate(secondChild);
+   TR::Register *src3Register = cg->evaluate(thirdChild);
+   TR::Register *targetRegister;
+
+   if(type == TR::Float)
+      targetRegister = cg->allocateSinglePrecisionRegister();
+   else
+   //DataType == TR::Double
```

Comment:
I don't think this comment is necessary.  Please remove.

Reply:
Fixed in 05e6a71